spec/features/auth_feature_spec.rb

```diff
@@ -9,7 +10,8 @@
     fill_in "Password", :with => "test"
     fill_in "Confirm password", :with => "test"
     check "user_tos"
-    UserMailer.should_receive(:deliver_activation_instructions)
+    UserMailer.should_receive(:activation_instructions).and_return(email)
```

Comment:
I noticed it's version 2.99. It's the version that deprecates the old syntax but brings in the new one to make converting easier.  In fact, @tylerritchie, if the tests are passing, just run [transpec](https://github.com/yujinakayama/transpec) on it and it will convert mostly everything for you.